modules/primer-marketing-utilities/docs/layout.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Layout
+status: New release
+status_issue: https://github.com/github/design-systems/issues/325
```

Comment:
We're moving away from status issues and plan to just let people open a new issue in the Primer repo so that it will work when this goes public.